### PR TITLE
Add jenkins_job states

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@bersace opm?

File: _states/jenkins_job.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+import difflib
+
+
+def present(name, xml_path, **kwargs):
```
Comment (bersace) [inline _states/jenkins_job.py]: What about using `source` argument name. Like state `file.managed` ?

**Comment by toopy** [comment]
@bersace up'

**Comment by toopy** [comment]
pas trop tôt :p

**Pull request merged**